src/features/search/components/CalendarPicker.tsx

```diff
@@ -0,0 +1,86 @@
+import { t } from '@lingui/macro'
+import DateTimePicker from '@react-native-community/datetimepicker'
+import React, { useState } from 'react'
+import { Platform } from 'react-native'
+import styled from 'styled-components/native'
+
+import { DateFilterModal } from 'features/offer/components/DateFilterModal'
+import { _ } from 'libs/i18n'
+import { formatToCompleteFrenchDate } from 'libs/parsers/formatDates'
+import { useModal } from 'ui/components/modals/useModal'
+import { ColorsEnum, getSpacing, Typo } from 'ui/theme'
+
+import { DateFilterButton } from '../atoms/DateFilterButton'
+
+export const CalendarPicker: React.FC = () => {
+  const [date, setDate] = useState(new Date(Date.now()))
+  const [mode, setMode] = useState<'time' | 'date' | undefined>('date')
+  const [show, setShow] = useState<boolean>(false)
+  const [isSelected, setIsSelected] = useState<boolean>(false)
+
+  const onChange = (event: Event, selectedDate: Date | undefined) => {
```

Comment:
```suggestion
  const onChange = (_event: Event, selectedDate: Date | undefined) => {
```

sinon tu vas avoir des warnings